extensions/rocksdb-repos/ProvenanceRepository.h

```diff
@@ -56,21 +55,13 @@ class ProvenanceRepository : public core::Repository, public std::enable_shared_
     db_ = NULL;
   }
 
-  // Destructor
-  virtual ~ProvenanceRepository() {
-    if (db_)
-      delete db_;
-  }
-
-  virtual void flush();
+  virtual void printStats();
```

Comment:
This new function is local to the class and doesn't need to be `virtual`.

Reply:
Removed.